src/components/styles/Icon.style.ts

```diff
@@ -1,6 +1,8 @@
 import styled from "styled-components";
-import text from '../../assets/text.png'
+import text from '../../assets/text.png';
 import password from '../../assets/password.png'
+import eye from '../../assets/eye.svg'
+import eyeHide from '../../assets/eyeHide.svg'
 import line from '../../assets/line.png'
```

Comment:
jeśli tsconfig jest dobrze skonfigurowany to nie trzeba dodawać ../../
wystarczy import line from 'assets/line.png'

Reply:
A rzeczywiście to okropnie wygląda, nie zwróciłam na to uwagi wcześniej. Przelecę się po aplikacji i spojrzę gdzie jeszcze to poprawić